packages-exp/auth-exp/src/model/auth.d.ts

```diff
@@ -31,6 +31,15 @@ export interface Config {
 }
 
 export interface Auth {
```

Comment:
Will the public types eventually be moved to `@firebase/auth-types-exp` ?

Reply:
Yeah, the files in the model/ directory are meant to contain internal stuff as well. The types in the types package will expose only the public bits (and many of the fields will be marked as readonly, for example)